src/js/config/socket.json

```diff
@@ -1,5 +1,5 @@
 {
-  "url": "wss://www2.binaryws.com/websockets/v3?app_id=1098&l=EN",
+  "url": "wss://www2.binaryws.com/websockets/v3?app_id=1098&l=JA",
```

Comment:
@jy-binary , After discussion with @ashkanx , I will keep this card as it is. There is another config file under config.production directory.
At the moment, Mohamed is working on making this configuration dynamic.